libraries/WiFi/src/WiFiScan.h

```diff
@@ -43,6 +43,7 @@ class WiFiScanClass
     wifi_auth_mode_t encryptionType(uint8_t networkItem);
     int32_t RSSI(uint8_t networkItem);
     uint8_t * BSSID(uint8_t networkItem);
+    uint8_t * BSSID(uint8_t networkItem, uint8_t* bssid);
```

Comment:
use just one API: `uint8_t * BSSID(uint8_t networkItem, uint8_t* bssid = NULL);`
and replace both
```
uint8_t * BSSID(uint8_t networkItem);
uint8_t * BSSID(uint8_t networkItem, uint8_t* bssid);
```